portal/src/main.rs

```diff
@@ -139,6 +141,8 @@ async fn main() -> Result<(), zbus::Error> {
         | zbus::fdo::RequestNameFlags::AllowReplacement;
     cnx.request_name_with_flags(PORTAL_NAME, flags).await?;
 
+    tracing::info!("Got dbus name {PORTAL_NAME}");
```

Comment:
what is this for ?

Reply:
Just to know that the binary reached the "main" loop without erring out, but it might be unnecessary.

Reply:
I removed it, if the process fails it will just close.